lib/README.md

```diff
@@ -0,0 +1,24 @@
+# `lib`
+
+This directory includes supplementary files for Markcop. This file includes an over of how to navigate and edit these.
```

Comment:
Good catch, just submitted https://github.com/hackclub/markcop/pull/108.

On Mon, Jun 13, 2016 at 6:57 PM, Max Wofford <notifications@github.com>
wrote:

> In lib/README.md
> <https://github.com/hackclub/markcop/pull/105#discussion_r66898908>:
>
> > @@ -0,0 +1,24 @@
> > +# `lib`
> > +
> > +This directory includes supplementary files for Markcop. This file includes an over of how to navigate and edit these.
>
> file includes an over of how
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/hackclub/markcop/pull/105/files/284b60339d20e3a8a777e8807dda01c3259eb7c4#r66898908>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA8j-IUlCjjbIWgZf9ii5OrCeUz53i3zks5qLgqFgaJpZM4IsDqU>
> .
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>